scripts/build-tokens.js

```diff
@@ -33,25 +33,16 @@ async function run() {
+    'yarn transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-global.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-brand-light.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-light.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-light.macos.json --out ./packages/theming/theme-tokens/src/generated/light-macos --p reactnative',
   );
-  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/global-macos/reactnative/tokens-aliases.json'));
-  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/global-macos/reactnative/tokens-controls.json'));
-
-  console.log('Generating macOS light mode tokens...');
-  child_process.execSync(
-    'yarn transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-light.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-light.macos.json --out ./packages/theming/theme-tokens/src/generated/light-macos --p reactnative',
-  );
-  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/light-macos/reactnative/tokens-global.json'));
   fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/light-macos/reactnative/tokens-controls.json'));
 
-  console.log('Generating macOS dark mode tokens...');
+  console.log('Generating macOS dark mode global and alias tokens...');
   child_process.execSync(
-    'yarn transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-dark.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-dark.macos.json --out ./packages/theming/theme-tokens/src/generated/dark-macos --p reactnative',
+    'yarn transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-global.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-brand-dark.macos.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-dark.json --in ./packages/theming/theme-tokens/src/pipeline-input/token-input-dark.macos.json --out ./packages/theming/theme-tokens/src/generated/dark-macos --p reactnative',
   );
-  fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/dark-macos/reactnative/tokens-global.json'));
   fs.unlinkSync(path.join(process.cwd(), 'packages/theming/theme-tokens/src/generated/dark-macos/reactnative/tokens-controls.json'));
```

Comment:
If this API matches the same one in python, anywhere we use `path.join()` we should be providing each directory as an arguement (`path.join('foo', 'bar', 'xyz')` instead of `path.join('foo', 'bar/xyz')`. That way, `Path.join()` will inject the correct path separators per OS ( "\" for windows, "/" for Darwin AKA macOS, idk for linux).

Reply:
Thanks for the suggestion, this would motivate us to get the script working on macOS? :)